tests/test_settings.py

```diff
@@ -173,6 +173,9 @@ def test_check_settings_wrong_setting_name(self):
         param('NORMALIZE', 'true', '', True),
         param('FUZZY', 'true', '', False),
         param('PREFER_LOCALE_DATE_ORDER', 'false', '', True),
+        param('DEFAULT_LANGUAGES', 'en', '', ['en']),
+        param('LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD', '1', '', 0.5),
+        param('DEFAULT_LANGUAGES', '1', '', ['en']),
```

Comment:
I've seen that you put the same setting twice. I would suggest to keep only the first one.
```suggestion
        param('DEFAULT_LANGUAGES', 'en', '', ['en']),
        param('LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD', '1', '', 0.5),
```